packages/bitcore-wallet-service/src/lib/expressapp.ts

```diff
@@ -67,13 +66,17 @@ export class ExpressApp {
       next();
     });
 
-    const POST_LIMIT = 1024 * 100 /* Max POST 100 kb */;
+    const POST_LIMIT = 1024 * 100; // Max POST 100 KB
+    const POST_LIMIT_LARGE = 2 * 1024 * 1024; // Max POST 2 MB
 
-    this.app.use(
-      bodyParser.json({
-        limit: POST_LIMIT
-      })
-    );
+    this.app.use((req, res, next) => {
```

Comment:
What about this?
`    this.app.use((req, res, next) => {
      const postLimit = req.path.includes('/txproposals') ? POST_LIMIT_LARGE : POST_LIMIT;
      return express.json({ limit: postLimit })(req, res, next);
    });`

Reply:
Can't figure out how to format it well. The suggestion is assigning a const with the limit of the ternary, & then just referencing it once w/in the express.json call.

Reply:
Use triple backticks for formatting